deploy/europe.bicep

```diff
@@ -415,3 +415,20 @@ resource cosmosDbDatabaseThroughput 'Microsoft.DocumentDB/databaseAccounts/sqlDa
     cosmosDbAccount
   ]
 }
+
+resource mySQLServer 'Microsoft.DBforMySQL/servers@2017-12-01' = {
+  name: '${resourceNamePrefix}-mysql-srv'
+  location: location
+  sku: {
+    name: 'B_Gen4_1'
```

Comment:
What is the cost of this?

Reply:
Updated the SKU as I saw there were a cheaper one.
The cost is calculated to $25.32 / month with updated SKU.

Reply:
Great, thanks!